spec/pronto/github_spec.rb

```diff
@@ -66,6 +61,8 @@ module Pronto
 
       context 'pull request does not exist' do
         specify do
+          ENV['PRONTO_PULL_REQUEST_ID'] = 'foo'
```

Comment:
I think the tests should not mutate global state to avoid random failures dependent on test order.
I'd either cache the old result:

```ruby
let(:old_env_var) { ENV['PRONTO_PULL_REQUEST_ID'] }
let(:env_var) { 'foo' }
before(:each) do
  old_env_var
  ENV['PRONTO_PULL_REQUEST_ID'] = env_var
end
after(:each) do
  ENV['PRONTO_PULL_REQUEST_ID'] = old_env_var
end
```

or use stubs:

```ruby
allow(ENV).to receive(:[]).with('PRONTO_PULL_REQUEST_ID').and_return('foo')
allow(ENV).to receive(:fetch) ....
```

Reply:
Used stubs like so: `ENV.stub(:[]).with('PRONTO_PULL_REQUEST_ID').and_return(10)`

Couldn't use `allow` as the "should syntax" is configured. 

Also had to stub other calls to `ENV` using `ENV.stub(:[])`. That's because ENV is called like 25 times in [Pronto::Config](https://github.com/prontolabs/pronto/blob/master/lib/pronto/config.rb#L10). This leads me to an idea: Wouldn't it be better to create method `Pronto::Config#pronto_pull_request_id` and use and stub that?